apps/root/src/pages/transfer/components/recipient-address/components/contact-item/index.tsx

```diff
@@ -178,11 +152,7 @@ const ContactItem = ({ contact, onClickContact, onDeleteContact }: ContactItemPr
                   <>
                     <FormattedMessage description="lastUpdated" defaultMessage="Last Updated" />
                     {': '}
-                    {DateTime.fromMillis(contact.label.lastModified).toLocaleString({
-                      month: 'long',
-                      day: 'numeric',
-                      year: 'numeric',
-                    })}
+                    {DateTime.fromMillis(contact.label.lastModified).toFormat('MMMM dd, yyyy')}
```

Comment:
DateTime.DATE_MED is what you want :)

Reply:
and use toLocaleString so it automatically updates once we change the language

Reply:
https://moment.github.io/luxon/#/formatting?id=presets

Reply:
Awesome! Thanks :D